user-guide/Advanced_Functionality/Databases/Elasticsearch_database/Installing_Elasticsearch_on_Linux.md

```diff
@@ -0,0 +1,116 @@
+
+1. Mount the data folder to the data disk.
+
+   > [!NOTE]
+   >
+   > - The folder where the Elasticsearch data is stored is configured in *elasticsearch.yaml*, in the property *path.data*.
+   > - To verify on which disk the data is mounted, execute the "df" command.
+
+1. Configure the user rights for the Elasticsearch user/group in the data folder.
+
+   You can do this with the following command:
+
+   `$ chown -R elasticsearch:elasticsearch /directoryname_from_path.data`
+
+1. Configure the *elasticsearch.yml* files.
+
+   - Make sure the Elasticsearch service is stopped. To stop the Elasticsearch service, use the following command:
+
+     `$ sudo systemctl stop elasticsearch.service`
+
+   - To change *elasticsearch.yml* files, you can use the following command:
+
+     `$ vi /etc/elasticsearch/elasticsearch.yml`
+
+   - Make the following adjustments in the *elasticsearch.yml* file based on your setup:
+
+     - **cluster.name**: This needs to be the same for all nodes in your Elasticsearch cluster.
+
+     - **node.name**: The name of the Elasticsearch node.
+
+     - **path.data**: The location(s) where you want to store the data.
+
+     - **bootstrap.memory_lock**: Set this to *true*.
+
+     - **network.host**: The IP address of the node.
+
+     - **http.port**: Set this to *9200*.
+
+     - **discovery.zen.ping.unicast.hosts**: The IP addresses of all the nodes in your Elasticsearch cluster. Elasticsearch nodes use this list of hosts to find each other and learn the topology of the ring.
+
+       > [!NOTE]
+       >
+       > - We recommend a cluster of three nodes, preferably in different racks.
+       > - To add a node to an existing cluster, see [Adding an Elasticsearch cluster node](xref:Configuring_Elasticsearch_node_add).
+
+     - **discovery.zen.minimum_master_nodes**: Set this to *2*. For more information on master nodes, see [Configuring the master nodes](xref:Configuring_master_Elasticsearch_nodes).
+
+     - **gateway.recover_after_nodes**: Set this to *1*.
+
+     - **node.master**: Set this to *true*. For more information on master nodes, see [Configuring the master nodes](xref:Configuring_master_Elasticsearch_nodes).
+
+1. Make the following adjustments in */etc/sysconfig/elasticsearch*:
+
+   - **MAX_LOCKED_MEMORY**: Set this to *unlimited*.
+
+   - **JAVA_HOME**: Set this to the custom Java path to be used for Elasticsearch, e.g. *java-11-openjdk-11.0.14.0.9-1.el7_9.x86_64*.
+
+1. Make the following adjustment in */usr/lib/systemd/system/elasticsearch.service*:
+
+   - **LimitMEMLOCK**: Set this to *infinity*. This must be set under the *Service* tag.
+
+1. Set the maximum Java Heap Size under */etc/elasticsearch/jvm.options*. For more information, see [Setting the heap size](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/heap-size.html).
+
```

Comment:
maybe add an !Important note here that Elasticsearch uses quite a bit of memory and that at least 8 Gb of Heap size should be specified, and more may be required depending on the demands of the system.